src/utils/get-package-base.ts

```diff
@@ -22,6 +24,5 @@ module.exports.getPackageName = function (id) {
       return pkgNameMatch[0].replace(/\\/g,  '/');
     }
   }
+  return undefined;
```

Comment:
There was an implicit return here so do we expect this to be undefined in some cases?

Reply:
Yes, the undefined result is used for the "invalid" case.